src/ui/flutter_app/lib/shared/custom_drawer/src/item.dart

```diff
@@ -58,16 +58,15 @@ class CustomDrawerItem<T> extends StatelessWidget {
           title,
           style: TextStyle(
             fontWeight: selected ? FontWeight.w700 : FontWeight.w500,
-            fontSize: LocalDatabaseService.display.fontSize,
             color: _color,
           ),
         ),
       ],
     );
 
     return InkWell(
-      splashColor: AppTheme.drawerSplashColor,
-      highlightColor: AppTheme.drawerHighlightColor,
+      splashColor: const Color(0x1a9e9e9e),
```

Comment:
I don't know why you changed it to a magic number.

Reply:
That's indeed a good question.
Thanks for catching that.

As said in my comment I'll go with something different anyways.

Reply:
OK. Properties, Attributes, Consts, these are alternative names.